store/sqlstore/session_store.go

```diff
@@ -105,25 +95,18 @@ func (me SqlSessionStore) Get(sessionIdOrToken string) (*model.Session, *model.A
 func (me SqlSessionStore) GetSessions(userId string) ([]*model.Session, *model.AppError) {
 	var sessions []*model.Session
 
-	tcs := make(chan store.StoreResult, 1)
-	go func() {
-		teams, err := me.Team().GetTeamsForUser(userId)
-		tcs <- store.StoreResult{Data: teams, Err: err}
-		close(tcs)
-	}()
 	if _, err := me.GetReplica().Select(&sessions, "SELECT * FROM Sessions WHERE UserId = :UserId ORDER BY LastActivityAt DESC", map[string]interface{}{"UserId": userId}); err != nil {
 		return nil, model.NewAppError("SqlSessionStore.GetSessions", "store.sql_session.get_sessions.app_error", nil, err.Error(), http.StatusInternalServerError)
 	}
 
-	rtcs := <-tcs
-	if rtcs.Err != nil {
-		return nil, model.NewAppError("SqlSessionStore.GetSessions", "store.sql_session.get_sessions.app_error", nil, rtcs.Err.Error(), http.StatusInternalServerError)
+	teams, err := me.Team().GetTeamsForUser(userId)
```

Comment:
ame here, `teamMembers` is more accurate

Reply:
done